### DISCUSSION TITLE
Audio Missing in Downloaded Livestream Video

### DISCUSSION THREAD

**Discussion by superb26** [aclap-dev/video-downloadhelper#1433] [Help and Q&A]
### Your question

I downloaded a live stream video, but there's no audio when I play it. Can you help me figure out why the sound isn't coming through and what I can do to fix this issue?

### Addon & CoApp version and details

In order to best help you, we need the most precise information you can share:

Your versions (VDH, CoApp, Operating System, Browser). If you don't know, there's an easy way, please read this.
A clear description of the problem
If appropriate, a public link to a video about the problem. We will not use your credentials to connect to non-public sites.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Did you actually read what it says there under **Addon & CoApp version and details**?  Read it.  Read it again.  And read it yet again.  Then DO WHAT IT SAYS.
